jx/jx-install.md

```diff
@@ -2,7 +2,71 @@
 
 ---
 
+# Install [doctl](https://github.com/digitalocean/doctl)
```

Comment:
This is in a wrong file. It should be in https://github.com/vfarcic/vfarcic.github.io/blob/master/jx/jx-cluster-tekton-demo.md. I already put it there.